### ISSUE TITLE
Bonn is broken.

### ISSUE THREAD

**Issue by jklmnn** [offenesdresden/ParkAPI#114]
Last available data for Bonn is from 2016-03-18T13:14:56.

**Comment by jklmnn** [comment]
Bonn has moved to http://bcp-bonn.de/ which currently doesn't seem scrapable.

**Comment by kiliankoe** [comment]
`$ curl http://bcp-bonn.de | grep "Freie Stell"`

```
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>094 <img src='/wp-content/uploads/2016/01/16.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>353 <img src='/wp-content/uploads/2016/01/07.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>676 <img src='/wp-content/uploads/2016/01/07.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>210 <img src='/wp-content/uploads/2016/01/07.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>368 <img src='/wp-content/uploads/2016/01/16.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
<h3><span style="color: #89bb24;">Freie Stellplätze: <strong>041 <img src='/wp-content/uploads/2016/01/16.png' width='19' height='19' style='vertical-align:middle'></strong></span></h3>
```

It'll have to be adjusted, but why not scrapable?

**Comment by jklmnn** [comment]
I didn't find that piece of code :D

**Comment by jklmnn** [comment]
Added fix with d3a19ff3c7afc363a9de15fa861b586c654eca65.
We still need to add the last_updated time.

**Comment by jklmnn** [comment]
How do we handle the last_updated if no information is provided at the source?

**Comment by sibbl** [comment]
I'd say we have at least these 3 solutions:

1. making `last_updated` optional, which would require some changes in the apps
2. using the `last_downloaded` value
3. calculating it by ourselves by checkin when the last value in the database changed

I think option 2 would be the easiest for now, what do you think? As I'm not familiar with the db: how complicated would the 3rd option be?

**Comment by jklmnn** [comment]
I'm for the 2. option.

**Comment by kiliankoe** [comment]
Iirc option 2 is already used in other cities as well. Should be fine.

**Comment by jklmnn** [comment]
Fixed this with 376f900dba2f3d9ef8e7d4ed2c252107787fff52.

**Comment by jklmnn** [comment]
Broken again.

**Comment by kiliankoe** [comment]
In the future we should just output via the API if some city is constantly changing their formats and present the users with a button to mail those responsible directly 🙈😄

**Comment by jklmnn** [comment]
Fixed with 1700f1cf784d551b7d917d0a85db67f1a1d8fd2e.
